src/containers/ResponseDisplay/SubmissionFiles.jsx

```diff
@@ -42,15 +39,22 @@ export class SubmissionFiles extends React.Component {
                   {
                     Header: 'Name',
                     accessor: 'name',
-                    Cell: this.renderHeaderEllipsesCell,
+                    Cell: FileNameCell,
+                  },
+                  {
+                    Header: 'File Extension',
+                    accessor: 'name',
+                    id: 'extension',
+                    Cell: FileExtensionCell,
                   },
                   {
-                    Header: 'Description',
-                    accessor: 'description',
-                    Cell: this.renderHeaderEllipsesCell,
+                    Header: 'Popover',
```

Comment:
```suggestion
                    Header: 'File Metadata',
```
The other 2 are their data purpose, not their element type.